package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
+    "bulma": "^0.1.2",
```

Comment:
@alexdiliberto @crodriguez1a Yes, I went through the dummy app and all looked good. Looking at the [Bulma CHANGELOG](https://github.com/jgthms/bulma/blob/master/CHANGELOG.md), seems like all releases since 0.1.2 were fixes and additions, no breaking changes. Interestedly though, in the 0.4.0 release the default font-size was changed to 16, that could be considered a breaking change for some.